cfgov/cfgov/settings/base.py

```diff
@@ -96,6 +96,7 @@
     'hmda',
     'youth_employment',
     'diversity_inclusion',
+    'mega_menu.apps.MegaMenuConfig',
```

Comment:
Somehow I never learned that the preferred format for entries in [`settings.INSTALLED_APPS`](https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-INSTALLED_APPS) is the dotted Python path to an application configuration class, not the a package containing an application.

Reply:
You are not alone here.